exams/tasks_test.py

```diff
@@ -60,6 +61,25 @@ def test_task_retry(self, task, task_retry):
         retry.assert_called_once_with(countdown=1, exc=error)
         return self
 
+    @data(
+        (export_exam_authorizations, 'export_exam_authorizations got an ImproperlyConfigured exception: %s'),
+        (export_exam_profiles, 'export_exam_profiles got an ImproperlyConfigured exception: %s')
+    )
+    @unpack
+    def test_task_improperly_config_logged(self, task, expected_warning_message):
+        """
+        Verify that when a retryable error occurs that the task retries
+        """
+        error = ImproperlyConfigured()
+        with patch("exams.tasks.log") as log, patch('exams.pearson.upload.upload_tsv') as upload_tsv_mock:
+            upload_tsv_mock.side_effect = error
+            task.delay()
+
+        log.warning.assert_called_with(
+            expected_warning_message, error
+        )
+        return self
```

Comment:
Not sure why this `return self` is here, except that it is in the test method above. Can you please remove both?